Makefile

```diff
@@ -11,7 +11,13 @@ BUILDDIR      = _build
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-.PHONY: help Makefile
+.PHONY: help Makefile lint
+
+lint:
+	pipenv run pycodestyle .
+	pipenv run flake8 .
+	pipenv run pydocstyle .
+	pipenv run mypy .
```

Comment:
The `Makefile` here is for the documentation build, so it might not be obvious that `make lint` exists at all (because nobody reads this `Makefile`). We should add a script in `Pipfile` that just calls `make lint`.

```
lint = "make lint"
```